src/main/java/com/direwolf20/buildinggadgets/common/schema/BoundingBox.java

```diff
@@ -35,6 +31,21 @@ public BoundingBox(Vec3i min, Vec3i max) {
         this(min.getX(), min.getY(), min.getZ(), max.getX(), max.getY(), max.getZ());
     }
 
+    public BoundingBox expand(int x, int y, int z) {
+        return new BoundingBox(minX - x, minY - y, minZ - z, maxX + x, maxY + y, maxZ + z);
+    }
+
+    /**
+     * @return A new Region who's max coordinates are increased by the given amount
+     */
+    public BoundingBox grow(int x, int y, int z) {
+        return new BoundingBox(minX, minY, minZ, maxX + x, maxY + y, maxZ + z);
+    }
+
+    public Stream<BlockPos> stream() {
```

Comment:
This is a welcomed improvement :+1: please close your other PR as this now modifies it. Or update your old Pr and I'll accept it then you can rebase your work ontop of that.

Reply:
I merged the old PR into here... Just merge #501

Reply:
Try to keep your PR's pure in the future.

Reply:
That's why I did the other one. I expected to delegate to to bounding box, so I merged that in, but found out that I don't need that just yet. Which is why it was valid imo to be here - I just expected the other PR to be merged way earlier.

Reply:
Still, attempt to keep your PR's pure.